mod/groups/start.php

```diff
@@ -22,6 +22,7 @@ function groups_init() {
 
 	// Set up the menu
 	$item = new ElggMenuItem('groups', elgg_echo('groups'), 'groups/featured');
+	$item->setTooltip('Groups');
```

Comment:
This ends up making it English only, should probably be an elgg_echo in there.

Reply:
Check https://github.com/gctools-outilsgc/gcconnex/pull/2543/commits/3ddd801f02188da8be34c67287fd2e9bb7a2ba05